nservicebus/upgrades/acs-host-7to8.md

```diff
@@ -12,7 +12,19 @@ upgradeGuideCoreVersions:
  - 7
 ---
 
-The NServiceBus Azure Host will be deprecated as of Version 9 and users are recommended to switch to self hosting for new endpoints. Upgrading existing endpoints is still supported for Version 8
+The NServiceBus Azure Host will be deprecated as of Version 9 and users are recommended to switch to self hosting for new endpoints. Upgrading existing endpoints is still supported for Version 8.
```

Comment:
Should we make this an include and also include into the host upgrade guide?

Reply:
Agreed, we need this in the windows host upgrade guide as well. 

That said some parts will differ since eg. usage of `SC.exe` isn't relevant when running in azure?

Reply:
Hm... I could omit the link and make it more generic.